src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -123,6 +128,7 @@ def import_config(cmd,
 
         src_kvs = __read_kv_from_config_store(src_azconfig_client,
                                               key=src_key,
+                                              snapshot=src_snapshot,
```

Comment:
Do we need to read feature flags from this snapshot?

Reply:
You are right. I missed this.  For snapshots we don't filter out any kvs. When a user specifies `--skip-keyvault` or `--skip-features` with snapshots, should we fail the command or just log a warning and import all kvs?

Reply:
I think we should throw an error. When importing/exporting from a snapshot, we dont allow any further filtering. 
@jimmyca15, thoughts?

Reply:
Agree on throwing an error.

Reply:
This has been updated accordingly